stdlib/private/StdlibCollectionUnittest/CheckCollectionType.swift.gyb

```diff
@@ -1549,24 +1573,61 @@ extension TestSuite {
     func toCollection(_ r: CountableRange<Int>) -> C {
       return makeCollection(r.map { wrapValue(OpaqueValue($0)) })
     }
+
+    self.test("\(testNamePrefix).index(after:)/semantics") {
```

Comment:
This whole section looks perfectly GYBbable. There are only two things that change between the mutating and non-mutating variants: test name and the line that produces the next index.